core/src/main/java/io/github/tcdl/msb/api/ObjectFactory.java

```diff
@@ -18,16 +17,6 @@
     Requester createRequester(String namespace, RequestOptions requestOptions);
 
     /**
-     * Creates a new instance of {@link Requester} with originalMessage.
-     *
-     * @param namespace       topic name to send a request to
-     * @param requestOptions  options to configure a requester
-     * @param originalMessage original message (to take correlation id from)
-     * @return instance of a requester
-     */
-    Requester createRequester(String namespace, RequestOptions requestOptions, Message originalMessage);
```

Comment:
I'm not sure that  removing method createRequester which uses originalMessage as parameter is good idea.